c/certificateService.c

```diff
@@ -173,7 +179,6 @@ static int serveMappingService(HttpService *service, HttpResponse *response)
         }
 
         userMapStructure.distinguishedNameLength = strlen(distinguishedId);
-        memset(userMapStructure.distinguishedName, 0, strlen(distinguishedId));
```

Comment:
I believe, the intent was to clear entire field with zeroes and then copy the field, which is redundant because of next line and fact, the null terminator is not used by the macro, which relies on the length fields only. Removing the line is OK.